gnome-initial-setup/pages/display/gis-display-page.c

```diff
@@ -72,41 +70,57 @@ set_toggle_options_from_config (GisDisplayPage *page)
 }
 
 static gboolean
-read_screen_config (GisDisplayPage *page)
+should_display_overscan (GisDisplayPage *page)
 {
   GisDisplayPagePrivate *priv = gis_display_page_get_instance_private (page);
-  GnomeRRConfig *current;
-  GnomeRROutputInfo **outputs;
-  GnomeRROutputInfo *output;
-  int i;
 
-  gnome_rr_screen_refresh (priv->screen, NULL);
+  if (!priv->current_output)
+    return FALSE;
+
+  return cc_display_monitor_supports_underscanning (priv->current_output);
+}
+
+static gboolean
```

Comment:
It does not look like you make any use of this return value.

Reply:
I will use it in the updated PR